### ISSUE TITLE
No Fabric Support?

### ISSUE THREAD

**Issue by LutraVibes** [TheCymaera/minecraft-spider#7]
The title is self-explanatory.

**Comment by HerrChaos** [comment]
The title is not self-explanatory. You mean you want this as a mod?

**Comment by mikidomikari** [comment]
At the very least it would be nice to not have to run a Paper server every time I want to use this mod. I own a server and I still can't get it to work. Would love to figure out how it really works and prank my friends with it.

**Comment by Vinfinity-Gaming** [comment]
Well, either way its server-side, so they still wont know, even if its a server or a fabric mod. Besides, took me only like 10 minutes to set up 1.21.


TIP: when you're getting your server up (you'll need the map for the thing unfortunately since the items are required) put the contents of the folder in the spider garden zip into the world folder after its created for the world to be used.

TIP 2: When you do get your server up with the proper map, the items are in the barrel next to the prismarine thing.

**Comment by LutraVibes** [comment]
> At the very least it would be nice to not have to run a Paper server every time I want to use this mod. I own a server and I still can't get it to work. Would love to figure out how it really works and prank my friends with it.

Yes, I agree. This is exactly what I'd like as well.

**Comment by LutraVibes** [comment]
> Well, either way its server-side, so they still wont know

Yeah, I'm aware of that. I already have a docker environment for fabric servers, and I don't want to have to set up a paper one.

**Comment by LutraVibes** [comment]
That's exactly why I want it to be a fabric mod as well

**Comment by HerrChaos** [comment]
Maybe ask on DC. No one checks the GitHub.
